### PR TITLE
Fix #744 : Highfi: Topic Practice

### LINKED ISSUE #744: Highfi: Topic Practice
TopicPractice needs to be pixel perfect with the mocks.
Portrait Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/f403add1-079a-4202-9975-efe04c076290/TP-Practice-Unchecked-
Landscape Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0fa47d7c-e10d-4097-8773-0efa665a323f/TP-Practice-

Mocks have been created for MDPI Devices.
Make sure that the screen passes the accessibility test also.
For reference you can checkout: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by virajpatel2411**
## Mock
Portrait Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/f403add1-079a-4202-9975-efe04c076290/TP-Practice-Unchecked-
Landscape Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0fa47d7c-e10d-4097-8773-0efa665a323f/TP-Practice-

## Explanation
UI updated as per mock, passes accessibility test

### CODE REVIEW

**Comment by virajpatel2411** [comment]
@rt4914 @veena14cs @nikitamarysolomanpvt  Can you please review this PR?

File: app/src/main/res/layout-land/topic_practice_footer_view.xml [hunk 1]
```diff
@@ -15,25 +15,24 @@
...
       android:text="@string/topic_practice_start"
-      android:textColor="@{isSubmitButtonActive ? @color/white : @color/submitButtonInactive}"
+      android:textColor="@{isSubmitButtonActive ? @color/white : @color/submitButtonTextInactive}"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout-land/topic_practice_footer_view.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout-land/topic_practice_header_view.xml [hunk 2]
```diff
@@ -34,12 +32,12 @@
...
+      android:layout_marginTop="4dp"
       android:fontFamily="sans-serif"
       android:text="@string/topic_practice_skills_description"
       android:textColor="@color/oppiaSecondaryText"
       android:textSize="16sp"
       app:layout_constraintStart_toStartOf="@+id/master_skills_text_view"
       app:layout_constraintTop_toBottomOf="@+id/master_skills_text_view" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout-land/topic_practice_header_view.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout-land/topic_practice_subtopic.xml [hunk 3]
```diff
@@ -31,20 +32,19 @@
...
+      android:drawableStart="@android:color/transparent"
+      android:drawablePadding="20dp"
+      android:minHeight="22dp"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout-land/topic_practice_subtopic.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout/topic_practice_footer_view.xml [hunk 4]
```diff
@@ -23,15 +23,16 @@
...
-      android:textColor="@{isSubmitButtonActive ? @color/white : @color/submitButtonInactive}"
+      android:layout_marginBottom="108dp"
+      android:textColor="@{isSubmitButtonActive ? @color/white : @color/submitButtonTextInactive}"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout/topic_practice_footer_view.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout/topic_practice_subtopic.xml [hunk 5]
```diff
@@ -46,4 +48,4 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout/topic_practice_subtopic.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout/topic_practice_header_view.xml [hunk 6]
```diff
@@ -21,18 +23,20 @@
...
       android:fontFamily="sans-serif"
       android:text="@string/topic_practice_skills_description"
       android:textColor="@color/oppiaSecondaryText"
       android:textSize="16sp"
       app:layout_constraintStart_toStartOf="@+id/master_skills_text_view"
-      app:layout_constraintTop_toBottomOf="@+id/master_skills_text_view"/>
+      app:layout_constraintTop_toBottomOf="@+id/master_skills_text_view" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout/topic_practice_header_view.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/values/colors.xml [hunk 7]
```diff
@@ -117,4 +119,4 @@
   <color name="avatar_background_22">#AB29CC</color>
   <color name="avatar_background_23">#CC29B1</color>
   <color name="avatar_background_24">#CC2970</color>
-</resources>
+</resources>
```
Comment (rt4914) [inline app/src/main/res/values/colors.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/values/colors.xml [hunk 8]
```diff
@@ -55,7 +55,9 @@
   <!-- SUBMIT BUTTON INACTIVE -->
   <color name="defaultAudioActivityRadioButton">#00A89F</color>
   <color name="submitButtonInactive">#61999999</color>
+  <color name="submitButtonTextInactive">#61000000</color>
   <color name="submitButtonInactiveText">#61000000</color>
+
```
Comment (rt4914) [inline app/src/main/res/values/colors.xml]: Remove this blank line

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -292,11 +293,13 @@
...
+
   <string name="topics_in_progress">Topics in Progress</string>
   <string name="topic_in_progress">Topic in Progress</string>
   <string name="stories_completed">Stories Completed</string>
   <string name="story_completed">Story Completed</string>
   <!-- WalkthroughActivity -->
   <string name="walkthrough_welcome_description">Learn new math skills with stories that show you how to use them in your daily life</string>
   <string name="welcome">"Welcome %s!"</string>
-</resources>
+</resources>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: EOF is missing, add one blank line at the end.

**Review by rt4914 — CHANGES_REQUESTED** [review]
EOF (End of File) is missing in all files. Add exactly one blank line in each file.

Otherwise the PR is perfect. Thanks.

File: app/src/main/res/values/strings.xml (hunk 9 above)
Comment (virajpatel2411) [reply app/src/main/res/values/strings.xml]: Added

File: app/src/main/res/values/colors.xml (hunk 8 above)
Comment (virajpatel2411) [reply app/src/main/res/values/colors.xml]: Done

File: app/src/main/res/values/colors.xml (hunk 7 above)
Comment (virajpatel2411) [reply app/src/main/res/values/colors.xml]: Done

File: app/src/main/res/layout/topic_practice_header_view.xml (hunk 6 above)
Comment (virajpatel2411) [reply app/src/main/res/layout/topic_practice_header_view.xml]: Done

File: app/src/main/res/layout/topic_practice_subtopic.xml (hunk 5 above)
Comment (virajpatel2411) [reply app/src/main/res/layout/topic_practice_subtopic.xml]: Done

File: app/src/main/res/layout/topic_practice_footer_view.xml (hunk 4 above)
Comment (virajpatel2411) [reply app/src/main/res/layout/topic_practice_footer_view.xml]: Done

File: app/src/main/res/layout-land/topic_practice_subtopic.xml (hunk 3 above)
Comment (virajpatel2411) [reply app/src/main/res/layout-land/topic_practice_subtopic.xml]: Done

File: app/src/main/res/layout-land/topic_practice_header_view.xml (hunk 2 above)
Comment (virajpatel2411) [reply app/src/main/res/layout-land/topic_practice_header_view.xml]: Done

File: app/src/main/res/layout-land/topic_practice_footer_view.xml (hunk 1 above)
Comment (virajpatel2411) [reply app/src/main/res/layout-land/topic_practice_footer_view.xml]: Done

File: .idea/codeStyles/Project.xml [hunk 10]
```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (rt4914) [inline .idea/codeStyles/Project.xml]: We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

**Comment by virajpatel2411** [comment]
@rt4914 @nikitamarysolomanpvt @veena14cs Done the required changes. Can you please merge this PR after a final check?

**Review by rt4914 — CHANGES_REQUESTED** [review]
One unnecessary change has been introduced.

Also, you have made a forcepush to one of the commits. Please NEVER do that. For this PR we are accepting it just considering that you have already put in a lot of efforts on this PR but in future PRs this won't be accepted.

**Comment by virajpatel2411** [comment]
> One unnecessary change has been introduced.
> 
> Also, you have made a forcepush to one of the commits. Please NEVER do that. For this PR we are accepting it just considering that you have already put in a lot of efforts on this PR but in future PRs this won't be accepted.

Okayy I won't do that in future

File: .idea/codeStyles/Project.xml (hunk 10 above)
Comment (virajpatel2411) [reply .idea/codeStyles/Project.xml]: Done

**Comment by virajpatel2411** [comment]
@rt4914 Done the changes. Reverted Project.xml

**Review by rt4914 — APPROVED** [review]
LGTM, thanks

**Pull request merged**